### PR TITLE
Force the prophecy version to 1.7.0

### PR DESCRIPTION
**Pull request by aRn0D**
**Description (for Contributor and Core Developer)**

This PR fix spec for 1.6

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Repository/ChannelRepository.php [hunk 1]
```diff
@@ -21,16 +21,24 @@ class ChannelRepository extends EntityRepository implements ChannelRepositoryInt
     /**
      * {@inheritdoc}
      */
-    public function findBy(array $criteria, array $orderBy = ['label' => 'ASC'], $limit = null, $offset = null)
+    public function findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
     {
+        if (null !== $orderBy) {
```
Comment (jjanvier) [inline src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Repository/ChannelRepository.php]: ===

no?

File: src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Repository/ChannelRepository.php [hunk 2]
```diff
@@ -21,16 +21,24 @@ class ChannelRepository extends EntityRepository implements ChannelRepositoryInt
...
         return parent::findBy($criteria, $orderBy, $limit, $offset);
     }
 
     /**
      * {@inheritdoc}
      */
-    public function findOneBy(array $criteria, array $orderBy = ['label' => 'ASC'])
+    public function findOneBy(array $criteria, array $orderBy = null)
     {
+        if (null !== $orderBy) {
```
Comment (jjanvier) [inline src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Repository/ChannelRepository.php]: ===

no?

File: src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Repository/LocaleRepository.php [hunk 3]
```diff
@@ -20,16 +20,24 @@ class LocaleRepository extends EntityRepository implements LocaleRepositoryInter
     /**
      * {@inheritdoc}
      */
-    public function findBy(array $criteria, array $orderBy = ['code' => 'ASC'], $limit = null, $offset = null)
+    public function findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
     {
+        if (null !== $orderBy) {
```
Comment (jjanvier) [inline src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Repository/LocaleRepository.php]: same

File: src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Repository/LocaleRepository.php [hunk 4]
```diff
@@ -20,16 +20,24 @@ class LocaleRepository extends EntityRepository implements LocaleRepositoryInter
...
         return parent::findBy($criteria, $orderBy, $limit, $offset);
     }
 
     /**
      * {@inheritdoc}
      */
-    public function findOneBy(array $criteria, array $orderBy = ['code' => 'ASC'])
+    public function findOneBy(array $criteria, array $orderBy = null)
     {
+        if (null !== $orderBy) {
```
Comment (jjanvier) [inline src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Repository/LocaleRepository.php]: same

File: src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Repository/ChannelRepository.php (hunk 1 above)
Comment (aRn0D) [reply src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Repository/ChannelRepository.php]: oup yes good catch

**Pull request merged**